src/components/queryBuilder/views/LogsQueryBuilder.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useState } from 'react';
+import { ColumnsEditor } from '../ColumnsEditor';
+import { BuilderMode, Filter, TableColumn, OrderBy, QueryBuilderOptions, SelectedColumn, ColumnHint } from 'types/queryBuilder';
+import { ColumnSelect } from '../ColumnSelect';
+import { Switch } from '../Switch';
+import { OtelVersionSelect } from '../OtelVersionSelect';
+import { OrderByEditor } from '../OrderByEditor';
+import { LimitEditor } from '../LimitEditor';
+import { FiltersEditor } from '../FilterEditor';
+import allLabels from 'labels';
+import { getColumnByHint } from 'components/queryBuilder/utils';
+
+interface LogsQueryBuilderProps {
+  allColumns: TableColumn[];
+  builderOptions: QueryBuilderOptions,
+  onBuilderOptionsChange: (builderOptions: QueryBuilderOptions) => void;
+}
+
+export const LogsQueryBuilder = (props: LogsQueryBuilderProps) => {
+  const { allColumns, builderOptions, onBuilderOptionsChange } = props;
+  const [otelEnabled, setOtelEnabled] = useState<boolean>(false);
+  const [otelVersion, setOtelVersion] = useState<string>('');
```

Comment:
I guess we need to use `enum` or union, since the version has a limited set of valid values

Reply:
For now I have a simple `.ts` file with an array of versions. Depending on how OTEL does their versions, we could make it an enum/union